source/interface/azure_iot_mqtt.h

```diff
@@ -20,6 +20,7 @@
 #include "azure_iot_mqtt_port.h"
 #include "azure_iot_transport_interface.h"
 
+#define azureiotmqttPACKET_TYPE_BIT_MASK       ( ( uint8_t ) 0xF0U )  /** @brief Mask to acquire the packet type */
```

Comment:
how about defining macro to get the packetType from mqtt control feild? 

#define azureiotmqttGET_PACKET_TYPE( ucType )       ( ucType  & 0xF0U )